source/_posts/2017-02-03-hassbian-toybox.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "HASSbian 1.1 - The Toy-box"
+description: "New release of the HASSbian Raspberry Pi Image for Home Assistant"
+date: 2017-02-04 05:00:00 -0400
+date_formatted: "February 4, 2017"
+author: Fredrik Lindqvist
+comments: true
+categories: Technology
+---
+
+Tonight I'm happy to announce a new release of the our Raspberry Pi image, **HASSbian 1.1 - The Toy-box.**  
+Why Toy-box you wonder? Because it encompass the changes pretty well.
+
+Changes from previous image are big and small but lets start with the interesting things.
+
+### {% linkable_title Hassbian-scrips %}
+A set of script written to add extra functionality to your raspberry pi installation. 
+This scripts are run as the `pi` user and installs a set of tools or packages.
+Currently includes:
+ - Install Libcec. Adds local [HDMI CEC support][cec].
+ - Install Mossquitto. Installs the latest Mosquitto package and client tools from the Mosquitto projects official repository. Now includes web-socket support.
```

Comment:
Later you use `websocket`. According to the RFC, the spelling is `WebSocket` but in our docs we use `websocket`.